### PR TITLE
Create Contributing.md

### PR DESCRIPTION
**Pull request by martincheck**
I'm submitting this pull request more for feedback than expecting this to be done.  I need to get the formatting a little cleaned up, and have a few TBA sections.  Feel free to bounce back with feedback, or fill in the blanks, or remove the TBA's and post as is (i.e. good enough and we'll iterate)

### CODE REVIEW

File: Contributing.md [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+We're looking for and eagerly accepting all contributions.  Whether you want to improve performance or fix bugs, or improve documentation,  triage bugs, or write  tutorials, any contribution is welcome.  
+
+
+Ground Rules
+<To be added>
+
+Your First Contribution
+
+Unsure where to begin contributing? You can start by looking through these beginner and help-wanted issues: 
+<to be added - some blurb about
```
Comment (maggiepint) [inline Contributing.md]: Typically, @mj1856 uses the up-for-grabs tag to indicate beginner issues.

File: Contributing.md [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+<to be added - some blurb about 
+Beginner issues - issues which should only require a few lines of code, and a test or two.
+Help wanted issues - issues which should be a bit more involved than beginner issues. 
+Both issue lists are sorted by total number of comments. While not perfect, number of comments is a reasonable proxy for impact a given change will have
+/TBA>
+
+Working on your first Pull Request? You can learn how from this free series, How to Contribute to an Open Source Project on GitHub.
+
+Getting started:
+For something that is bigger than a one or two line fix:
```
Comment (maggiepint) [inline Contributing.md]: In this project we would probably fork for everything.

**Comment by mattjohnsonpint** [comment]
Thanks for getting this started!  Let's keep this here in pull-request form for the time being.  We can continue to discuss and you can simply push additional commits to your branch for them to show up here.

Perhaps start with adding URLs where appropriate?  That would help. 😄

File: Contributing.md [hunk 3]
```diff
@@ -0,0 +1,70 @@
...
+* What operating system and processor architecture are you using?
+* What did you do?
+* What did you expect to see?
+* What did you see instead?
+
+## How to suggest a feature or enhancement
+As noted in the introduction, SQL Server 2016 includes built-in support for Windows time zones using a new AT TIME ZONE syntax. If you only need support for Windows time zones (not IANA time zones), consider using this feature instead of this project. Read [the news here](http://blogs.technet.com/b/dataplatforminsider/archive/2015/11/30/sql-server-2016-community-technology-preview-3-1-is-available.aspx), and documentation [here](https://msdn.microsoft.com/en-us/library/mt612795.aspx).  
+
+## Code review process
+Explain how a contribution gets accepted after it’s been submitted.
```
Comment (maggiepint) [inline Contributing.md]: Let's remove the boilerplate here.

File: Contributing.md [hunk 4]
```diff
@@ -0,0 +1,70 @@
...
+Who reviews it? Who needs to sign off before it’s accepted? When should a contributor expect to hear from you? How can contributors get commit access, if at all?
+
+The core team (basically Matt) looks at Pull Requests on a best effort basis. We seek to get better, and are looking for volunteers to co-maintain the library to increase the cadence.
```
Comment (maggiepint) [inline Contributing.md]: This is pretty accurate

File: Contributing.md [hunk 5]
```diff
@@ -0,0 +1,70 @@
+# Introduction:
+First off, thank you for considering contributing to SQL Server Time Zone Support.  If you're passionate about Time zones, passionate about SQL Server, or you just want to make the SQL world a little better, you're in the right place.  
+
+Following these guidelines helps to communicate that you respect the time of the developers managing and developing this open source project. In return, the maintainers will reciprocate that respect in addressing your issue, assessing changes, and helping you finalize your pull requests.
+
+We're looking for and eagerly accepting all contributions.  Whether you want to improve performance or fix bugs, or improve documentation,  triage bugs, or write  tutorials, any contribution is welcome.  
+
+
+## Ground Rules
```
Comment (maggiepint) [inline Contributing.md:9]: Let's either remove this, or replace it with a link to a code of conduct like [Contributor covenant](https://www.contributor-covenant.org/version/1/4/code-of-conduct.md)

File: Contributing.md [hunk 6]
```diff
@@ -0,0 +1,70 @@
...
+> Content to be added
+
+## Your First Contribution
+
+Unsure where to begin contributing? You can start by looking through these beginner and help-wanted issues:
```
Comment (maggiepint) [inline Contributing.md]: I would simplify this to "Unsure where to begin contribution? Look for issues with an up-for-grabs or help-wanted tag"

**Review by maggiepint — CHANGES_REQUESTED** [review]
This is in pretty good shape. Clean up these couple things and I'll merge it

File: Contributing.md (hunk 5 above)
Comment (martincheck) [reply Contributing.md:9]: any preference regrading whether I copy and paste this inline, or link to it?  I'll start with linking and see how that works...

**Review by maggiepint — APPROVED** [review]

**Pull request merged**